generator/src/routes/+page.svelte

```diff
@@ -29,30 +81,57 @@
   <div class="form-wrapper">
     <div class="title">Generate a product passport</div>
     <img class="logomarkSvg" src={LogomarkSvg} alt="" aria-hidden="true" />
-    <form>
+    <form on:submit|preventDefault={onGenerate}>
       <div class="row">
-        <FormInputGroup name="domain" label="Issuer domain" placeholder="ex.tags.ioxio.dev" />
+        <FormInputGroup
+          name="iss"
+          label="Issuer domain"
+          placeholder="ex.tags.ioxio.dev"
+          disabled={status === Status.GENERATING || isValid}
+          required
+        />
       </div>
       <div class="row">
         <div class="toggle-row">
-          <Toggle options={["Arbitrary", "Premade"]} bind:value={productOption} />
+          <Toggle
+            options={["Arbitrary", "Premade"]}
+            bind:value={productOption}
+            disabled={status === Status.GENERATING}
+          />
         </div>
         <FormSelectGroup
           name="product"
           label="Product"
           placeholder="Type a product"
           options={data.options}
+          disabled={status === Status.GENERATING}
+          required
         />
```

Comment:
It is not possible for me to enter any product I want. The expected default was "Arbitrary", as in that you can just enter any value you want.